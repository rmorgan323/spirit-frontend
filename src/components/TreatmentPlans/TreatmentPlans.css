/******  GLOBAL CSS VARIABLES  ******/
/***  COLORS  ***/
/***  FONTS  ***/
* {
  font-size: 16px;
  font-family: "Lato", sans-serif;
  outline: none;
  color: #6d6e71; }

h1, h2, h3, h4, h5, h6 {
  font-family: "Josefin Sans", sans-serif;
  line-height: 1.3; }

h1 {
  font-size: 44px; }

h2 {
  font-size: 30px; }

h3 {
  font-size: 24px; }

h4 {
  font-size: 18px; }

h5 {
  font-size: 14px; }

h6 {
  font-size: 10px; }

p {
  font-size: 14px;
  line-height: 1.3;
  font-weight: 900; }

::placeholder {
  opacity: .5;
  text-align: center; }

/***  MIXINS  ***/
.TreatmentPlans {
  width: 875px;
  display: block;
  margin: 0 auto 50px; }
  .TreatmentPlans .treatment-header-tabs {
    display: flex; }
  .TreatmentPlans h4 {
    padding: 9px;
    margin: 0;
    text-align: center; }
  .TreatmentPlans .treatment-header {
    width: 175px;
    background-color: #a7a9ac;
    cursor: pointer;
    display: flex;
    align-items: center;
    justify-content: center;
    transition: all .2s ease-in-out;
    border-left: 1px solid #fff;
    border-right: 1px solid #fff; }
  .TreatmentPlans .treatment-header-modulation {
    border-left: 0; }
  .TreatmentPlans .treatment-header-executive {
    border-right: 0; }
  .TreatmentPlans .treatment-header h4 {
    color: #d1d3d4; }
  .TreatmentPlans .treatment-header:hover h4 {
    color: #6d6e71; }
  .TreatmentPlans .treatment-header:hover {
    background-color: #d1d3d4; }
  .TreatmentPlans .treatment-header-active {
    background-color: #d1d3d4; }
    .TreatmentPlans .treatment-header-active h4 {
      color: #6d6e71; }
  .TreatmentPlans .display-none {
    display: none; }
  .TreatmentPlans .module-container {
    background-color: #d1d3d4;
    padding: 10px; }
