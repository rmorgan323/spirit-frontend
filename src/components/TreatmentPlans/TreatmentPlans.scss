@import '../../variables.scss';

.TreatmentPlans {
  width: 875px;
  display: block;
  margin: 0 auto 50px;

  .treatment-header-tabs {
    display: flex;
  }

  h4 {
    padding: 9px;
    margin: 0;
    text-align: center;
  }

  .treatment-header {
    width: 175px;
    background-color: $grey3;
    cursor: pointer;
    display: flex;
    align-items: center;
    justify-content: center;
    @include std-transition;
    border-left: 1px solid #fff;
    border-right: 1px solid #fff;
  }

  .treatment-header-modulation {
    border-left: 0;
  }

  .treatment-header-executive {
    border-right: 0;
  }

  .treatment-header h4 {
    color: $grey2;
  }

  .treatment-header:hover h4 {
    color: $grey5;
  }

  .treatment-header:hover {
    background-color: $grey2;
  }

  .treatment-header-active {
    background-color: $grey2;

    h4 {
      color: $grey5;
    }
  }

  .display-none {
    display: none;
  }

  .module-container {
    background-color: $grey2;
    padding: 10px;
  }

} 