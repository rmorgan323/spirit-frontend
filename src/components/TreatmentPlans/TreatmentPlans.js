import React, { Component } from 'react';
import TreatmentPlanModule from '../../containers/TreatmentPlanModule/TreatmentPlanModule';
import './TreatmentPlans.css';

class TreatmentPlans extends Component {
  constructor() {
    super();

    this.state = {
      currentTab: 'modulation'
    };
  }

  handleTabChange = name => {
    this.setState({ currentTab: name });
  };

  render() {
    const { currentTab } = this.state;
    const tabHeaderTitles = [
      'Sensory Modulation',
      'Postural',
      'Sensory Discrimination',
      'Social/Emotional Development',
      'Executive Functioning'
    ];

    const tabHeaderClass = [
      'modulation',
      'postural',
      'sensory',
      'social',
      'executive'
    ];

    const treatmentModuleTypes = [
      'modulation_',
      'postural_',
      'sensory_discrimination_',
      'social_emotional_',
      'executive_functioning_'
    ];

    const tabHeaders = tabHeaderClass.map((className, index) => {
      return (
        <div
          key={index}
          className={
            currentTab === className
              ? `treatment-header treatment-header-active treatment-header-${className}`
              : `treatment-header treatment-header-${className}`
          }
          onClick={() => this.handleTabChange(className)}
        >
          <h4>{tabHeaderTitles[index]}</h4>
        </div>
      );
    });

    const treatmentModules = treatmentModuleTypes.map((type, index) => {
      return (
        <div
          key={index}
          className={
            currentTab === tabHeaderClass[index]
              ? 'treatment-body'
              : 'treatment-body display-none'
          }
        >
          <TreatmentPlanModule type={type} />
        </div>
      );
    });

    return (
      <div className="TreatmentPlans">
        <div className="treatment-header-tabs">{tabHeaders}</div>

        <div className="module-container">{treatmentModules}</div>
      </div>
    );
  }
}

export default TreatmentPlans;
