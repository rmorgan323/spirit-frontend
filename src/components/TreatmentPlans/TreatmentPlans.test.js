import React from 'react';
import TreatmentPlans from './TreatmentPlans';
import { shallow } from 'enzyme';

describe('TreatmentPlans tests', () => {
  it('Should match snapshot', () => {
    const renderedTreatmentPlans = shallow(<TreatmentPlans />);

    expect(renderedTreatmentPlans).toMatchSnapshot();
  });

  it('Should change state when a new tab is clicked', () => {
    const renderedTreatmentPlans = shallow(<TreatmentPlans />);
    const tabHeaders = [
      'postural',
      'sensory',
      'social',
      'executive',
      'modulation'
    ];

    expect(renderedTreatmentPlans.state().currentTab).toEqual('modulation');

    tabHeaders.forEach(tab => {
      renderedTreatmentPlans.instance().handleTabChange(tab);
      expect(renderedTreatmentPlans.state().currentTab).toEqual(tab);
    });
  });
});
