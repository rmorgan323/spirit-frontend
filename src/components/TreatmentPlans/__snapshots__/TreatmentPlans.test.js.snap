// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreatmentPlans tests Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TreatmentPlans />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="treatment-header-tabs"
        >
          <div
            className="treatment-header treatment-header-active treatment-header-modulation"
            onClick={[Function]}
          >
            <h4>
              Sensory Modulation
            </h4>
          </div>
          <div
            className="treatment-header treatment-header-postural"
            onClick={[Function]}
          >
            <h4>
              Postural
            </h4>
          </div>
          <div
            className="treatment-header treatment-header-sensory"
            onClick={[Function]}
          >
            <h4>
              Sensory Discrimination
            </h4>
          </div>
          <div
            className="treatment-header treatment-header-social"
            onClick={[Function]}
          >
            <h4>
              Social/Emotional Development
            </h4>
          </div>
          <div
            className="treatment-header treatment-header-executive"
            onClick={[Function]}
          >
            <h4>
              Executive Functioning
            </h4>
          </div>
        </div>,
        <div
          className="module-container"
        >
          <div
            className="treatment-body"
          >
            <Connect(TreatmentPlanModule)
              type="modulation_"
            />
          </div>
          <div
            className="treatment-body display-none"
          >
            <Connect(TreatmentPlanModule)
              type="postural_"
            />
          </div>
          <div
            className="treatment-body display-none"
          >
            <Connect(TreatmentPlanModule)
              type="sensory_discrimination_"
            />
          </div>
          <div
            className="treatment-body display-none"
          >
            <Connect(TreatmentPlanModule)
              type="social_emotional_"
            />
          </div>
          <div
            className="treatment-body display-none"
          >
            <Connect(TreatmentPlanModule)
              type="executive_functioning_"
            />
          </div>
        </div>,
      ],
      "className": "TreatmentPlans",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="treatment-header treatment-header-active treatment-header-modulation"
              onClick={[Function]}
            >
              <h4>
                Sensory Modulation
              </h4>
            </div>,
            <div
              className="treatment-header treatment-header-postural"
              onClick={[Function]}
            >
              <h4>
                Postural
              </h4>
            </div>,
            <div
              className="treatment-header treatment-header-sensory"
              onClick={[Function]}
            >
              <h4>
                Sensory Discrimination
              </h4>
            </div>,
            <div
              className="treatment-header treatment-header-social"
              onClick={[Function]}
            >
              <h4>
                Social/Emotional Development
              </h4>
            </div>,
            <div
              className="treatment-header treatment-header-executive"
              onClick={[Function]}
            >
              <h4>
                Executive Functioning
              </h4>
            </div>,
          ],
          "className": "treatment-header-tabs",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Sensory Modulation
              </h4>,
              "className": "treatment-header treatment-header-active treatment-header-modulation",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sensory Modulation",
              },
              "ref": null,
              "rendered": "Sensory Modulation",
              "type": "h4",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Postural
              </h4>,
              "className": "treatment-header treatment-header-postural",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Postural",
              },
              "ref": null,
              "rendered": "Postural",
              "type": "h4",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "2",
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Sensory Discrimination
              </h4>,
              "className": "treatment-header treatment-header-sensory",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sensory Discrimination",
              },
              "ref": null,
              "rendered": "Sensory Discrimination",
              "type": "h4",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "3",
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Social/Emotional Development
              </h4>,
              "className": "treatment-header treatment-header-social",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Social/Emotional Development",
              },
              "ref": null,
              "rendered": "Social/Emotional Development",
              "type": "h4",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "4",
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Executive Functioning
              </h4>,
              "className": "treatment-header treatment-header-executive",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Executive Functioning",
              },
              "ref": null,
              "rendered": "Executive Functioning",
              "type": "h4",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="treatment-body"
            >
              <Connect(TreatmentPlanModule)
                type="modulation_"
              />
            </div>,
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="postural_"
              />
            </div>,
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="sensory_discrimination_"
              />
            </div>,
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="social_emotional_"
              />
            </div>,
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="executive_functioning_"
              />
            </div>,
          ],
          "className": "module-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "host",
            "props": Object {
              "children": <Connect(TreatmentPlanModule)
                type="modulation_"
              />,
              "className": "treatment-body",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "type": "modulation_",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "host",
            "props": Object {
              "children": <Connect(TreatmentPlanModule)
                type="postural_"
              />,
              "className": "treatment-body display-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "type": "postural_",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "2",
            "nodeType": "host",
            "props": Object {
              "children": <Connect(TreatmentPlanModule)
                type="sensory_discrimination_"
              />,
              "className": "treatment-body display-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "type": "sensory_discrimination_",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "3",
            "nodeType": "host",
            "props": Object {
              "children": <Connect(TreatmentPlanModule)
                type="social_emotional_"
              />,
              "className": "treatment-body display-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "type": "social_emotional_",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "4",
            "nodeType": "host",
            "props": Object {
              "children": <Connect(TreatmentPlanModule)
                type="executive_functioning_"
              />,
              "className": "treatment-body display-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "type": "executive_functioning_",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="treatment-header-tabs"
          >
            <div
              className="treatment-header treatment-header-active treatment-header-modulation"
              onClick={[Function]}
            >
              <h4>
                Sensory Modulation
              </h4>
            </div>
            <div
              className="treatment-header treatment-header-postural"
              onClick={[Function]}
            >
              <h4>
                Postural
              </h4>
            </div>
            <div
              className="treatment-header treatment-header-sensory"
              onClick={[Function]}
            >
              <h4>
                Sensory Discrimination
              </h4>
            </div>
            <div
              className="treatment-header treatment-header-social"
              onClick={[Function]}
            >
              <h4>
                Social/Emotional Development
              </h4>
            </div>
            <div
              className="treatment-header treatment-header-executive"
              onClick={[Function]}
            >
              <h4>
                Executive Functioning
              </h4>
            </div>
          </div>,
          <div
            className="module-container"
          >
            <div
              className="treatment-body"
            >
              <Connect(TreatmentPlanModule)
                type="modulation_"
              />
            </div>
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="postural_"
              />
            </div>
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="sensory_discrimination_"
              />
            </div>
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="social_emotional_"
              />
            </div>
            <div
              className="treatment-body display-none"
            >
              <Connect(TreatmentPlanModule)
                type="executive_functioning_"
              />
            </div>
          </div>,
        ],
        "className": "TreatmentPlans",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="treatment-header treatment-header-active treatment-header-modulation"
                onClick={[Function]}
              >
                <h4>
                  Sensory Modulation
                </h4>
              </div>,
              <div
                className="treatment-header treatment-header-postural"
                onClick={[Function]}
              >
                <h4>
                  Postural
                </h4>
              </div>,
              <div
                className="treatment-header treatment-header-sensory"
                onClick={[Function]}
              >
                <h4>
                  Sensory Discrimination
                </h4>
              </div>,
              <div
                className="treatment-header treatment-header-social"
                onClick={[Function]}
              >
                <h4>
                  Social/Emotional Development
                </h4>
              </div>,
              <div
                className="treatment-header treatment-header-executive"
                onClick={[Function]}
              >
                <h4>
                  Executive Functioning
                </h4>
              </div>,
            ],
            "className": "treatment-header-tabs",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "0",
              "nodeType": "host",
              "props": Object {
                "children": <h4>
                  Sensory Modulation
                </h4>,
                "className": "treatment-header treatment-header-active treatment-header-modulation",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sensory Modulation",
                },
                "ref": null,
                "rendered": "Sensory Modulation",
                "type": "h4",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "host",
              "props": Object {
                "children": <h4>
                  Postural
                </h4>,
                "className": "treatment-header treatment-header-postural",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Postural",
                },
                "ref": null,
                "rendered": "Postural",
                "type": "h4",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "2",
              "nodeType": "host",
              "props": Object {
                "children": <h4>
                  Sensory Discrimination
                </h4>,
                "className": "treatment-header treatment-header-sensory",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sensory Discrimination",
                },
                "ref": null,
                "rendered": "Sensory Discrimination",
                "type": "h4",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "3",
              "nodeType": "host",
              "props": Object {
                "children": <h4>
                  Social/Emotional Development
                </h4>,
                "className": "treatment-header treatment-header-social",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Social/Emotional Development",
                },
                "ref": null,
                "rendered": "Social/Emotional Development",
                "type": "h4",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "4",
              "nodeType": "host",
              "props": Object {
                "children": <h4>
                  Executive Functioning
                </h4>,
                "className": "treatment-header treatment-header-executive",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Executive Functioning",
                },
                "ref": null,
                "rendered": "Executive Functioning",
                "type": "h4",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="treatment-body"
              >
                <Connect(TreatmentPlanModule)
                  type="modulation_"
                />
              </div>,
              <div
                className="treatment-body display-none"
              >
                <Connect(TreatmentPlanModule)
                  type="postural_"
                />
              </div>,
              <div
                className="treatment-body display-none"
              >
                <Connect(TreatmentPlanModule)
                  type="sensory_discrimination_"
                />
              </div>,
              <div
                className="treatment-body display-none"
              >
                <Connect(TreatmentPlanModule)
                  type="social_emotional_"
                />
              </div>,
              <div
                className="treatment-body display-none"
              >
                <Connect(TreatmentPlanModule)
                  type="executive_functioning_"
                />
              </div>,
            ],
            "className": "module-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "0",
              "nodeType": "host",
              "props": Object {
                "children": <Connect(TreatmentPlanModule)
                  type="modulation_"
                />,
                "className": "treatment-body",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "type": "modulation_",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "host",
              "props": Object {
                "children": <Connect(TreatmentPlanModule)
                  type="postural_"
                />,
                "className": "treatment-body display-none",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "type": "postural_",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "2",
              "nodeType": "host",
              "props": Object {
                "children": <Connect(TreatmentPlanModule)
                  type="sensory_discrimination_"
                />,
                "className": "treatment-body display-none",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "type": "sensory_discrimination_",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "3",
              "nodeType": "host",
              "props": Object {
                "children": <Connect(TreatmentPlanModule)
                  type="social_emotional_"
                />,
                "className": "treatment-body display-none",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "type": "social_emotional_",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "4",
              "nodeType": "host",
              "props": Object {
                "children": <Connect(TreatmentPlanModule)
                  type="executive_functioning_"
                />,
                "className": "treatment-body display-none",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "type": "executive_functioning_",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
